zap/src/test/java/org/zaproxy/zap/utils/ByteBuilderTest.java

```diff
@@ -0,0 +1,241 @@
+package org.zaproxy.zap.utils;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ByteBuilderTest {
+
+    @Rule
+    public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testEnsureCapacity() {
+        ByteBuilder byteBuilder = new ByteBuilder(5);
+        Assert.assertEquals(5, byteBuilder.capacity());
```

Comment:
This is already being tested, please, add the (new) tests to existing class.

Reply:
`ByteBuilderUnitTest.java`

Reply:
Moved test to `ByteBuilderUnitTest.java`